Documentation/ApiOverview/ContentElements/_Migration/_ext_localconf.php.diff

```diff
@@ -0,0 +1,13 @@
+ use T3docs\Examples\Controller\FalExampleController;
+ use TYPO3\CMS\Extbase\Utility\ExtensionUtility;
+
+ ExtensionUtility::configurePlugin(
+     'Examples',
+     'FalExamples',
+        'HtmlParser',
+        [
+            HtmlParserController::class => 'index',
+        ],
++       [],
++    ExtensionUtility::PLUGIN_TYPE_CONTENT_ELEMENT,
```

Comment:
Something is completely broken here. 3rd Argument is an Array, but here it is "HtmlParser" and the indents of 6th (should be 5th) argument is missing 3 spaces. And in general the indents in PHP are 4 and not 7.

Reply:
This is a diff, not a PHP file?